### ISSUE TITLE
Disable move to current list

### ISSUE THREAD

**Issue by GoogleCodeExporter** [patel22p/mytinytodo#19]
```
When you click on the menu of a task, you are able to change the tasks list...
it's a great feature! (it should maybe also be in the advanced edit scren... 
oups I 
digress... :] )...

I think we shouldn't let the user choose the current list to move to... so i've 
made 
again a quick patch....


(PS: should i continue to submit 1 ticket / 1 issue / 1patch or should i make a 
list 
and post all in once?)

Thanks for your nice work :)
```

Original issue reported on code.google.com by `merz...@gmail.com` on 25 Jan 2010 at 10:49

Attachments:
- [disable_currentlist.patch](https://storage.googleapis.com/google-code-attachments/mytinytodo/issue-19/comment-0/disable_currentlist.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, please continue :)

I think - one issue for one feature (bug).
```

Original comment by `maxpozd...@gmail.com` on 26 Jan 2010 at 7:32

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c5cd1a96b5.
```

Original comment by `maxpozd...@gmail.com` on 26 Jan 2010 at 8:38
- Changed state: **Fixed**
